app/helpers/rivalries_helper.rb

```diff
@@ -11,7 +11,7 @@ module RivalriesHelper
     'connect your foot with their butt'
     'connect their butt with your foot'
     'kick kick kick kick kick kick'
-  ].freeze
+  ".freeze
```

Comment:
Does this work? `BUTT_KICK_SENTENCES` should be an array of strings, but this looks like it's one big string now.

Reply:
`%q[]` - it's not an array of strings
we need to use `%w[]` for array of strings :)

![image](https://user-images.githubusercontent.com/15526722/44307058-81a77700-a3a4-11e8-90be-72db62de4690.png)

Reply:
Oh wow, I guess this never worked then haha. It actually looks like the only method we use this in (`next_butt_kick_sentence` below) is never called, so let's actually just kill this whole file then.